sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -81,9 +81,9 @@ export interface DecryptResult {
 // @public
 export interface DeletedKey {
     id?: string;
-    key?: JsonWebKey;
+    key?: JsonWebKey_2;
```

Comment:
We solved this in the master branch by pinning api extractor to the version `7.7.11`

Reply:
I actually added that api-extractor fix, then removed it since I didn't think this differences would matter. I will add it back.

Reply:
Done!

Reply:
@sadasant In hot fix branches we should strive to do the minimum possible. With https://github.com/Azure/azure-sdk-for-js/pull/8782/commits/cabdfa212efa07fd529ed4b224aa9e203fb3b7c8, we are now pinning the api extractor for all the packages and so increasing the surface area of the PR.

Let's pin for only for the keyvault packages. If when running build, other *.api.md files are being edited, please undo those changes before committing changes to this PR

Reply:
Another option is to not pin the api-extractor dependency at all and not commit the changes to any of the *api.md files.

Reply:
I can't not do the pin only for some packages since rush requires consistency, unless there's some way of bypassing that that I'm not aware of. I will take your other option: to not pin the api-extractor dependency, and to not commit the changes to *api.md

Reply:
You can add an alternative version in `common\config\rush\common-versions.js`

Reply:
Ohh! Thank you Jeremy, I didn't know that. I picked the second option though, I've rebased and removed these two commits from history. There are no more api-extractor related changes in this PR>

Reply:
Seems like api-extractor has different levels of tolerance in different environments. This is failing on MacOS with Node12: https://dev.azure.com/azure-sdk/public/_build/results?buildId=385949&view=logs&j=b4852f1b-cbff-5fc1-4b9a-0828dbc40956&t=732f653e-38e4-5202-b2c0-8523cb504841&l=754 but not in any other environment (as far as the build is going). I will re-do the change and introduce the exception as Jeremy is describing.

Reply:
After I re-ran the build, it passed! So, disregard. I'm not adding these commits again.